.github/workflows/codespell.yml

```diff
@@ -17,5 +17,9 @@ jobs:
         uses: codespell-project/actions-codespell@master
         with:
           skip: .git,go.mod,go.sum,vendor
+          # kubebuilder latest release has a label with name kuberentes
+          # TODO: remove this in next kubebuilder update as this is fixed
+          # in kubebuilder master
+          ignore_words_list: kuberentes
```

Comment:
please add a link to the kubebuilder issue in this comment, that makes it easier to verify what next release contains the fix

Reply:
there was no issue created for it, it was fixed in a PR https://github.com/kubernetes-sigs/kubebuilder/pull/2964 here